### ISSUE TITLE
Time Picker Granularity

### ISSUE THREAD

**Issue by ddgrimmson418** [mominsamir/smDateTimeRangePicker#117]
Hi there, Its an awsm plug in that you have created however it lacks a functionality that might come in handy... as a developer I have seen a lot of date and time picker but one thing all of them lack is the resolution of the time is min by min which is not required and one should be able to customize the same... Any ways I am looking for a quick way to how to change the granularity so kindly help me in the same if you can.

**Comment by mominsamir** [comment]
when u say min time u want to add seconds to time picker?

**Comment by ddgrimmson418** [comment]
No.... what I mean is the granularity... I mean the time in the mins is displayed as 1,2,3,4,5.... 60 I want it in 5,10,15,20..... 60 ... I hope this clears

**Comment by tcasey** [comment]
@ddgrimmson418 You can change the the last part of the for loop for the `TimePickerCtrl.prototype.buildMinuteCells` That's where it creates the minute cells so if you want every 5 minutes for example just change that line to `for (var i = 0 ; i <= 59; i+=5) {`

**Comment by ddgrimmson418** [comment]
@tcasey @mominsamir  what trevor suggests basically means I will be changing the bower component code which is not a best practice in programming... can you please suggest a way it can be done without changing the bower component code
